sklearn/metrics/cluster/supervised.py

```diff
@@ -740,11 +739,7 @@ def adjusted_mutual_info_score(labels_true, labels_pred,
 
     """
     if average_method == 'warn':
-        warnings.warn("The behavior of AMI will change in version 0.22. "
-                      "To match the behavior of 'v_measure_score', AMI will "
-                      "use average_method='arithmetic' by default.",
-                      FutureWarning)
-        average_method = 'max'
+        average_method = 'arithmetic'
```

Comment:
you should remove this one as well. If some tests are failing because of this new default, you should fix `average_method='max'` for these specific tests.